test/e2e/virtual/apiexport/virtualworkspace_test.go

```diff
@@ -87,6 +88,20 @@ func TestAPIExportVirtualWorkspace(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 1, len(cowboysProjected.Items))
 
+	// TODO(ncdc): revisit when ThingPermissionClaim is in
+	t.Logf("Ensuring the appropriate core/v1 resources are available")
+	coreV1GVRs := []schema.GroupVersionResource{
+		{Version: "v1", Resource: "namespaces"},
+		{Version: "v1", Resource: "configmaps"},
+		{Version: "v1", Resource: "secrets"},
+		{Version: "v1", Resource: "serviceaccounts"},
+	}
+	for _, coreV1GVR := range coreV1GVRs {
+		t.Logf("Trying to wildcard list %q", coreV1GVR)
+		_, err = dynamicClients.Cluster(logicalcluster.Wildcard).Resource(coreV1GVR).List(ctx, metav1.ListOptions{})
```

Comment:
I think we really want to use the virtual cluster client here to make sure it is looking in the virtual workspace?

Reply:
Yes, thanks, that's what I get for trying to be super quick!

Reply:
Updated in latest force push - PTAL!